utils/zerovec/derive/examples/make_var.rs

```diff
@@ -142,6 +145,22 @@ where
     for (stack, zero) in slice.iter().zip(deserialized.iter()) {
         assert(stack, zero)
     }
+
+    if let Some(first) = varzerovec.get(0) {
+        let bincode = bincode::serialize(first).unwrap();
+        let deserialized: &T = bincode::deserialize(&bincode).unwrap();
```

Comment:
Suggestion: test `bincode::deserialize` into a `Box<T>` so that the code path is covered